pypeman/tests/test_channel.py

```diff
@@ -311,14 +370,47 @@ def test_memory_message_store(self):
         self.assertEqual(dict_msg['state'], 'error', "Message %s should be in error state!" % msg5)
 
 
+    def test_replay_from_memory_message_store(self):
+        """ We can store a message in FileMessageStore """
+
+        store_factory = msgstore.MemoryMessageStoreFactory()
+
+        chan = BaseChannel(name="test_channel10.5", loop=self.loop, message_store_factory=store_factory)
+
+        n = TestNode()
+
+        msg = generate_msg()
+        msg2 = generate_msg(timestamp=(1982, 11, 27, 12, 35))
+
+
+        chan.add(n)
+
+        # Launch channel processing
+
+        self.start_channels()
+        self.loop.run_until_complete(chan.handle(msg))
+        self.loop.run_until_complete(chan.handle(msg2))
+
+        msg_stored = list(chan.message_store.search())
+
+        for msg in msg_stored:
+            print(msg)
+
+        self.assertEqual(len(msg_stored), 2, "Should be 3 messages in store!")
+
+        self.loop.run_until_complete(chan.replay(msg_stored[0]['id']))
+
+        msg_stored = list(chan.message_store.search())
+        self.assertEqual(len(msg_stored), 3, "Should be 3 messages in store!")
+
     def test_file_message_store(self):
         """ We can store a message in FileMessageStore """
```

Comment:
same comment in previous test. 
It should be different, as the first line of a test docstring id displayed during testing.
Easier to find the correct function if unique